content/analyst/sfmc/competitive-tracker-installation-guide.md

```diff
@@ -0,0 +1,62 @@
+---
+lastUpdated: "02/23/2023"
+title: Competitive Tracker Installation Guide
+description: This guide helps Salesforce Marketing Cloud admins install SparkPost Competitive Tracker application through the AppExchange
+---
+
+If you are a Salesforce Marketing Cloud Administrator, please follow these one-time steps to install the SparkPost Competitive Tracker application through the AppExchange.
+
+### Step 1. Navigate to the AppExchange installation page by clicking [here](https://mc.exacttarget.com/cloud/#app/Administration/InstalledPackages/8283ba88-2804-4250-bf1d-e1e2b643747a/install)
```

Comment:
This link works but is a bit odd since it doesn't format the link in the heading 
![image](https://user-images.githubusercontent.com/10710273/222541357-c07de319-1220-44ab-be32-cf483f8316fe.png)